templates/registration/register.html.twig

```diff
@@ -0,0 +1,43 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Register{% endblock %}
+
+{% block stylesheets %}
+    {{ encore_entry_link_tags('login_style') }}
+{% endblock %}
+
+{% block body %}
+    <div class="container">
+        <div class="row justify-content-center sign-container">
+            <div class="col-xs-12 col-sm-6 col-md-4">
+                <h1 class="text-center login-title">Registration</h1>
+                <div class="account-wall">
+                    <img class="profile-img" src="{{ asset('build/images/login/add-user-icon-256.png') }}"
+                         alt="">
+                    {% for flashError in app.flashes('verify_email_error') %}
+                        <div class="alert alert-danger" role="alert">{{ flashError }}</div>
+                    {% endfor %}
+
+                    {{ form_start(registrationForm, {'attr': {'class': 'form-signin'}}) }}
+                    {{ form_widget(registrationForm.firstName, {'attr': {'class': 'form-control',
```

Comment:
Use `form_row` instead of `form_widget`, it displays errors.
Also, add line identation between `form_start` and `form_end`.

Reply:
Resolve

Reply:
@RomaSerenkov Line identation it is not new line, it is tabulation.

Reply:
Resolve